plugins/katello/3.3/release_notes/release_notes.md

```diff
@@ -4,16 +4,26 @@ title: Release Notes
 version: 3.3
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.3 (Baltic Porter) Release Notes
 
-{% if page.version and page.version != '3.3' %}
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
 
 ## Features
 
+### Updated UI
+
+### Improved Manifest Import and Delete
+
+### Username and Password for Repositories
+
+### Content View Version Promotion to Multiple Environments
+
+### Publish "Latest" Version in Composite Content Views
+
+### Capsule Download Policy
```

Comment:
Actually this should be called "Smart Proxy Content".  In fact we should probably include this as a feature so its more clear. 

Smart Proxies with content can now be configured with their own download policy.  This allows the main server to sync the full contents of its configured repositories while the Smart Proxy can be configured to use an "On Demand" policy and only sync what is actually used by clients.